ci/pipeline.yml

```diff
@@ -1629,11 +1773,14 @@ jobs:
       passed: [samus-cf-for-k8s]
     - get: capi-nginx
       passed: [samus-cf-for-k8s]
+    - get: backup-metadata
+      passed: [ backup-metadata-deploy-and-test ]
   - task: bump-capi-k8s-release-image-refs
     file: capi-ci/ci/k8s/bump-capi-k8s-release-image-refs.yml
   - put: capi-k8s-release
     params:
       repository: updated-capi-k8s-release
+      rebase: true
```

Comment:
I kept running into issues here with arbitrary (non `images.yml`) commits to capi-k8s-release causing this job to fail. I think this `rebase` here is safe and fine to do, but let me know if this is mistaken.

Reply:
I don't think that rebase is necessary now, since this job is the only thing that should be altering the contents of the branch.

Reply:
Instead, we should probably make the `k8s-ci-passed` job `serial: true`, so there's no race condition

Reply:
The ci-passed update actually happens after:
https://github.com/cloudfoundry/capi-ci/pull/15/files/a7bf34f72f28f4ec694bc1b80ea8b6995d51c695#diff-1d0bbbf28914b9bc860a34ad16f16af0a6996287a6757589f82da847185a8c68R1786

This is the piece that updates the "main" branch of the repro.

Reply:
Although if this rebases I wonder if that makes the main branch and the ci-passed branch look different.

Reply:
I added the `serial: true` cause that sounds good regardless.
And removed the `rebase: true` cause this probably won't be too big of a deal usually.

https://github.com/cloudfoundry/capi-ci/pull/15/commits/4cbec41fd7fa56cb0eee08c1fb4978a8a419a4d6